lib/jekyll-mentions.rb

```diff
@@ -15,7 +14,7 @@ def mentionify(doc)
         src = mention_base(doc.site.config)
         if doc.output.include? BODY_START_TAG
           parsed_doc    = Nokogiri::HTML::Document.parse(doc.output)
-          body          = parsed_doc.at_css('body')
+          body          = parsed_doc.at_css("body")
           body.children = filter_with_mention(src).call(body.inner_html)[:output].to_s
           doc.output    = parsed_doc.to_html
         else
```

Comment:
To get Rubocop to not yell at us about the method, you could move the contents of the first half of the condition to a private `parse_as_html` or whatever method (or you can ignore it for now).